test/integration/controlplane/synthetic_controlplane_test.go

```diff
@@ -191,6 +195,19 @@ func TestStatus(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		// test case 403 need an unprivileged user, specifically setting that up
+		if tc.name == "403" {
+			privilegedLoopbackToken := uuid.New().String()
+			// wrap any available authorizer
+			tokens := make(map[string]*user.DefaultInfo)
+			tokens[privilegedLoopbackToken] = &user.DefaultInfo{
+				Name:   user.APIServerUser,
+				UID:    uuid.New().String(),
+				Groups: []string{user.SystemPrivilegedGroup},
+			}
+			tokenAuthenticator := authenticatorfactory.NewFromTokens(tokens, tc.controlPlaneConfig.GenericConfig.Authentication.APIAudiences)
+			tc.controlPlaneConfig.GenericConfig.Authentication.Authenticator = authenticatorunion.New(tokenAuthenticator, authauthenticator.RequestFunc(framework.AlwaysEmpty))
+		}
```

Comment:
drop this and override the authenticator in initStatusForbiddenControlPlaneConfig instead to return an unprivileged user

Reply:
done